src/SpoolmanManager.cpp

```
@@ -1652,6 +1557,116 @@ int SpoolmanManager::findVendorNoLock(const char* name, char* outName, size_t ou
     return streamFindVendorByName(name, outName, outNameSize);
 }
 
+bool SpoolmanManager::fetchSpoolCore(int32_t spoolId, SpoolCore& out) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)